checks/runtimeCheck.go

```diff
@@ -28,13 +31,14 @@ func latestAgentTag(r *runtimeConfig) error {
 		CheckRedirect: func(req *http.Request, via []*http.Request) error {
 			return http.ErrUseLastResponse
 		},
+		Timeout: time.Second * 10,
 	}
 	resp, err := client.Get(r.agentVersionUrl)
 	if err != nil {
 		return err
 	}
 	defer resp.Body.Close()
-	if resp.StatusCode > 302 {
+	if resp.StatusCode < 300 || resp.StatusCode == 302 {
```

Comment:
I think I mislead you in my last comment. My mistake. This has us treating a success (200) as a failure.

Reply:
whoops, I meant to change it to `resp.StatusCode > 300 || resp.StatusCode == 302`